crates/cli/src/cli/mod.rs

```diff
@@ -31,8 +31,10 @@ pub const DEVNET_RPC_URL: &str = "https://api.devnet.solana.com";
 pub const TESTNET_RPC_URL: &str = "https://api.testnet.solana.com";
 pub const DEFAULT_ID_SVC_URL: &str = "https://id.txtx.run/v1";
 pub const DEFAULT_AUTH_SVC_URL: &str = "https://auth.txtx.run";
-pub const DEFAULT_CONSOLE_URL: &str = "https://txtx.run";
+pub const DEFAULT_CONSOLE_URL: &str = "https://cloud.txtx.run";
 pub const DEFAULT_SVM_GQL_URL: &str = "https://svm-cloud.gql.txtx.run/v1/graphql";
+pub const DEFAULT_SVM_CLOUD_API_URL: &str =
+    "https://3zsafgw57plgwwvv3ddt5mucnu0nioih.lambda-url.us-east-1.on.aws/v1/surfnets";
```

Comment:
Will this be a permanent url, or do we expect to change? Should this be an env? or behind a prettier url? Is the backend url something that shouldn't be available publicly?